### PR TITLE
6491 - Datagrid Render Only One Row

### PR DESCRIPTION
**Pull request by tjamesallen15**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

This PR fixed a bug where it renders all rows in the datagrid when adding one row.

**Related github/jira issue (required)**:
Closes https://github.com/infor-design/enterprise/issues/6491

**Steps necessary to review your pull request (required)**:
- Pull this branch, build, and run the app
- Go to http://localhost:4000/components/datagrid/test-hide-pager-if-one-page.html
- Add a `row`

**Included in this Pull Request**:
- [ ] An e2e or functional test for the bug or feature.
- [x] A note to the change log.

### CODE REVIEW

**Comment by tjamesallen15** [comment]
This is still in progress

File: src/components/datagrid/datagrid.js [hunk 1]
```diff
@@ -525,12 +525,54 @@ Datagrid.prototype = {
...
+    self.afterRender();
+  },
+
   /**
   * Add a row of data to the grid and dataset.
   * @param {object} data A data row object
   * @param {string} location Where to add the row. This can be 'bottom' or 'top', default is top.
   */
   addRow(data, location) {
+    console.log(data);
```
Comment (tmcconechy) [inline src/components/datagrid/datagrid.js:575]:
```suggestion
```

**Review by deep7102 — APPROVED** [review]

**Review by tmcconechy — DISMISSED** [review]

**Comment by tmcconechy** [comment]
@tjamesallen15 seems like a test is failing now

**Comment by tjamesallen15** [comment]
> @tjamesallen15 seems like a test is failing now

Checking.

**Pull request merged**